src/main/java/net/vulkanmod/vulkan/VRenderSystem.java

```diff
@@ -34,13 +35,13 @@ public abstract class VRenderSystem {
     public static int colorMask = PipelineState.ColorMask.getColorMask(true, true, true, true);
 
     public static boolean cull = true;
-
+    private static boolean canApplyClear = false;
```

Comment:
This should probably be should ;)

Reply:
Renamed it to `allowFullColorClear` instead if thats OK, which is a more accurate description and should be less misleading

Reply:
You that looks good